orcid-integration-test/src/test/java/org/orcid/integration/blackbox/web/account/PublicProfileVisibilityTest.java

```diff
@@ -36,7 +36,6 @@
 import org.openqa.selenium.By.ById;
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.interactions.Actions;
```

Comment:
Better to do a seperate pull request. For this unit of work I did:
https://github.com/ORCID/ORCID-Source/pull/2983